src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -2308,7 +2400,9 @@ private StatementAst IfStatementRule(Token ifToken)
                     // ErrorRecovery: assume the next token is a newline or part of something else,
                     // so stop parsing the statement and try parsing something else.
 
-                    ReportIncompleteInput(rParen.Extent, () => ParserStrings.MissingStatementBlock, keyword.Text);
+                    ReportIncompleteInput(rParen.Extent,
+                        nameof(ParserStrings.MissingStatementBlock),
+                        keyword.Text);
```

Comment:
Errors like this one could easily happen in other places in a PR with so many changes. I suggest you to go through your changes again and make sure this is the only case.

Reply:
Ah, the assertion I suggested would catch this as well - assuming we ran a test build and test all errors :)

Reply:
Yeah, I suspected this would be a risk. I'm intending to go back through and check all the changes before any merge. But I've included that assertion too to be surer.